nipype/interfaces/freesurfer/preprocess.py

```diff
@@ -950,14 +989,24 @@ def _is_resuming(self):
     def _format_arg(self, name, trait_spec, value):
         if name == 'T1_files':
             if self._is_resuming():
-                return ''
+                return None
```

Comment:
Switched to returning `None` just because that avoids the extra spaces in the `cmdline` string, making the doctests less painful to debug.